webthing/server.py

```diff
@@ -341,6 +342,51 @@ def check_origin(self, origin):
         """Allow connections from all origins."""
         return True
 
+    def update(self):
+        """Receive update from a Thing."""
+        pass
+
+    def update_property(self, property_):
+        """
+        Receive update from a Thing about an Property.
```

Comment:
Just a nitpick about the comments on these three methods... They should probably say `Send an update about X`, rather than `Receive`, given that this is at the server layer.

Reply:
I used "Receive" since the Subscriber receives an update while a Thing (the subject) sends the notification according to the [Observer pattern](https://refactoring.guru/design-patterns/observer/python/example). Then again a Thing uses the method to send an update to the Subscriber, so I'll change that.